CHANGELOG.md

```diff
@@ -1,15 +1,81 @@
 ## v2.0.0-beta.38 (2017-02-28)
 
+📦 🐈 Yarn support and more!
+
 #### :rocket: Enhancement
 * [#605](https://github.com/lerna/lerna/pull/605) Add support for pluggable npm clients. ([@gigabo](https://github.com/gigabo))
-* [#620](https://github.com/lerna/lerna/pull/620) Feature: skip-temp-tag. ([@noherczeg](https://github.com/noherczeg))
+
+> We'll make yarn the default once we feel that it's more stable.
+
+Usage:
+
+```sh
+$ lerna bootstrap --npm-client=yarn
+```
+
+```json
+{
+  "npmClient": "yarn"
+}
+```
+
+* [#595](https://github.com/lerna/lerna/pull/595) Publish npm packages in topological order ([@loganfsmyth](https://github.com/loganfsmyth))
+
+Very important fix for Babel that we used in the last release. This prevents a timing issue when publishing where a module will try to download a package that isn't published yet because it is published before it's own dependency is published itself. We used to get many issues from users on non-public npm about "babel-type" not being found.
+
 * [#608](https://github.com/lerna/lerna/pull/608) Add a --stream option to the run command. ([@gigabo](https://github.com/gigabo))
+
+Useful to get output for child processes immediately if using `lerna run` with a watch command
+
+```sh
+$ lerna run watch --stream
+```
+
 * [#624](https://github.com/lerna/lerna/pull/624) Add versions to lerna ls. Closes [#603](https://github.com/lerna/lerna/issues/603) ([@ben-eb](https://github.com/ben-eb))
+
+
+
+* [#620](https://github.com/lerna/lerna/pull/620) Feature: skip-temp-tag. ([@noherczeg](https://github.com/noherczeg))
+
+This will not create a temporary dist-tag called `lerna-temp` when publishing. Useful if your third party proxy doesn't support dist-tags.
+
+```sh
+$ lerna publish --skip-temp-tag
+```
+
 * [#587](https://github.com/lerna/lerna/pull/587) Always run test and env scripts. ([@simon360](https://github.com/simon360))
+
+Defaults to running `npm run test` and `npm run env` 
+
 * [#598](https://github.com/lerna/lerna/pull/598) Durable `includeFilteredDependencies` config via lerna.json. ([@gigabo](https://github.com/gigabo))
+
+```json
+{
+  "commands": {
+    "bootstrap" {
```

Comment:
I always just use `js` instead of `json` formatting for code blocks.  It's more forgiving if I mess something little like that up. 😅 

json:
```json
{
    ...
    "foo" {
       "bar": "baz",
    }
}
```

js:
```js
{
    ...
    "foo" {
       "bar": "baz",
    }
}
```